scripts/cipublish

```diff
@@ -1,37 +0,0 @@
-#!/bin/bash
-set -e
-
-if [[ -n "${DB_DEBUG}" ]]; then
-    set -x
-fi
-
-if [[ -n "${TRAVIS_COMMIT}" ]]; then
-    GIT_COMMIT="${TRAVIS_COMMIT:0:7}"  
-else
-    GIT_COMMIT=$(git rev-parse --short HEAD)
-fi
-
-function usage() {
-    echo -n \
-"Usage: $(basename "$0")
-Deploy container images to remote repository.
-"
-}
-
-
-if [ "${BASH_SOURCE[0]}" = "${0}" ]
-then
-    if [ "${1:-}" = "--help" ]
-    then
-        usage
-    else
-
-    docker login -u "${QUAY_USER}" -p "${QUAY_PASSWORD}" quay.io
-
-    docker push "quay.io/azavea/districtbuilder-nginx:${GIT_COMMIT}"
```

Comment:
Reminder to delete these images from Quay once they're no longer being used, or create an issue so that we can remember to do it later.

Reply:
Created #547